data/DataType/int_vlog.yaml

```diff
@@ -0,0 +1,4 @@
+﻿keyword: int
```

Comment:
We already have `int_c` for this.

There's no need to have duplicates. You can use the older files as they've the exact same thing.

If you want you can rename the `int_c` but it's ok if you don't. `int_c` is used in cpp too.

Reply:
fixed. 👍